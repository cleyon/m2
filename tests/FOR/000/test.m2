@for FOO 1 20 3
FOO = @FOO@
@next FOO
