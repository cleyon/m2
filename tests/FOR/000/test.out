FOO = 1
FOO = 4
FOO = 7
FOO = 10
FOO = 13
FOO = 16
FOO = 19
